wiki/simulation/simulating-vehicle-using-autoware.md

```diff
@@ -0,0 +1,112 @@
+---
+date: 2020-12-07
+title: Simulating Vehicles Using Autoware
+published: true
+---
+## Simulating and controlling an Ackermann drive chassis
+
+### Requirements
+- ROS (higher kinetic)
+- gazebo (higher version 7)
+
+Autoware is based on ROS and uses Gazebo for simulation. The goal of this section is to the the Gazebo simulator with Autoware packages. Autoware can use all the functionalities of ROS, it can be seen as another software layer over ROS. To simulate an ackermann vehicle we need to define an URDF or Xacro model. In this file we can change vehicle models, add actuators and sensors.
+Autoware offers a default xacro file which can be customized for use, this is all available forin the `vehicle_sim package`. To customize this model we need not change the xacro file. The vehicle dynamics can be changed in the config file `vehicle_model/config/caibration.yaml`. To customize the visual appearance of the vehicle we can use a custom mesh file. Collate (`.dae`) formats can be added to mesh folder and referenced in the vehicle.xacro file. We have later discussed how sensors can be added and customized.
+
+![](../../assets/images/team_j_wiki_automodels.png)*Figure 1. Different vehicle models*
```

Comment:
All supporting images need to be moved to wiki/simultion/assets. All links need to drop the `../..` and just be `![](assets/images/...)`.

Please also drop the `team_j_`, for something like `auotware_`.

For some reason this comment didn't post during my original review.  Otherwise, you're ready to merge!